weldx/utility.py

```diff
@@ -1039,10 +1037,16 @@ def time_ref_unset(self) -> xr.DataArray:
     def time_ref_restore(self) -> xr.DataArray:
         """Convert DatetimeIndex back to TimedeltaIndex + reference Timestamp."""
         da = self._obj.copy()
-        time_ref = da.weldx.time_ref
-        if time_ref and is_datetime64_dtype(da.time):
+        if "time" not in da.coords:
+            return da
+
+        if is_datetime64_dtype(da.time):
+            time_ref = da.weldx.time_ref
+            if time_ref is None:
+                time_ref = pd.Timestamp(da.time.data[0])
```

Comment:
This line isn't tested. Can we add a quck test?

Reply:
I will think of a test for these utility functions